backend/src/main/java/com/google/sps/FindMatchQuery.java

```diff
@@ -14,42 +14,32 @@
 
 package com.google.sps;
 
-import com.google.gson.Gson;
 import com.google.sps.data.Match;
 import com.google.sps.data.Participant;
-import java.time.Clock;
-import java.time.Duration;
-import java.time.Instant;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Date;
-import java.util.Collection;
-import java.util.Collections;
+import java.time.ZonedDateTime;
 import java.util.List;
-import java.util.stream.Collectors;
 
 /** Class used to find a match in a list of Participants with the most recently added Participant */
 public final class FindMatchQuery {
 
-  private Clock clock;
-  private static int MAX_DURATION_DIFF = 15; // maximum difference in duration to be compatible
-  private static int PADDING_TIME = 15; // extra padding time to ensure large enough meeting time block
-  
-  /** Constructor */
-  public FindMatchQuery() {
-    clock = new Clock();
-  }
+  /** Reference date time for "current" time */
+  private ZonedDateTime dateTime;
+  /** Maximum difference in duration to be compatible */
+  private static int MAX_DURATION_DIFF = 15;
+  /** extra padding time to ensure large enough meeting time block */
+  private static int PADDING_TIME = 15;
```

Comment:
static constants should be declared at the top of the class, before any instance fields